manifests/hub/spokeclusters.crd.yaml

```diff
@@ -0,0 +1,133 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: spokeclusters.cluster.open-cluster-management.io
```

Comment:
same with this CRD. See this for instance: https://github.com/openshift/cluster-config-operator/blob/master/Dockerfile#L10